libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergPropsBuilder.kt

```diff
@@ -20,7 +20,7 @@ data class GutenbergPropsBuilder(
     private val postType: String,
     private val featuredImageId: Int,
     private val editorTheme: Bundle?,
-    private val canViewEditorOnboarding: Boolean
+    val canViewEditorOnboarding: Boolean
```

Comment:
This can stay `private` now, right?

Reply:
Yes, you are correct. Good catch!

Reply:
Address in https://github.com/wordpress-mobile/WordPress-Android/pull/14766/commits/dcc546e678435e7ee80a2d093074ca83f89d9789.